LoopKitUI/Extensions/UIColor+LoopKit.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIColor.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 7/1/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+import UIKit
+
+extension UIColor {
```

Comment:
This extension is added so that the `CorrectionRangeOverridesEditor` can have access to the colors it needs